modules/nw-ovn-ipsec-encryption.adoc

```diff
@@ -4,8 +4,8 @@
 
 :_content-type: CONCEPT
 [id="nw-ovn-ipsec-encryption_{context}"]
-= Encryption protocol and tunnel mode for IPsec
+= Encryption protocol and IPsec mode
 
 The encrypt cipher used is `AES-GCM-16-256`. The integrity check value (ICV) is `16` bytes. The key length is `256` bits.
 
-The IPsec tunnel mode used is _Transport mode_, a mode that encrypts end-to-end communication.
+The IPsec mode used is _Transport mode_, a mode that encrypts end-to-end communication by adding an Encapsulated Security Payload (ESP) header to the IP header of original packet and encrypts the packet data. {product-title} does not currently use and support IPsec _Tunnel mode_ for pod-to-pod communication.
```

Comment:
Change "and" to "or" in "...use and support IPsec..."

Reply:
Addressed